src/custom-components/utils/customComponentsProps.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import * as Chakra from '@chakra-ui/react'
+
+import { ButtonProps } from '@chakra-ui/react'
+
+type PropsWithForm<T> = T & { form?: T }
+
+type PreviewDefaultProps = {
+  CC?: PropsWithForm<ButtonProps>
+}
+
+export const DEFAULT_PROPS: PreviewDefaultProps = {
```

Comment:
seems like duplicating existing code, try to fit this in existing DEFAULT_PROPS

Reply:
> seems like duplicating existing code, try to fit this in existing DEFAULT_PROPS

Made a different file since other custom components that will be created would add their default props in this file only and wouldn't unnecessarily re-render default Props for other components. This file can be completely removed and all the defaultProps can be managed in the already existing file as well making it easier to manage everything at one place. Performance difference should be negligible in both the scenarios.